src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -1162,6 +1162,8 @@
     <PropertyGroup  Condition="'$(AllowPartialPackages)' == 'true' AND '@(_missingPackageFile)' != ''">
       <Description>WARNING: This package is missing files @(_missingPackageFile->'%(TargetPath)/%(Filename)%(Extension)') %0A$(Description)</Description>
     </PropertyGroup>
+
+    <Error Condition="'$(LicenseUrl)' == ''" Text="LicenseUrl property is not defined." />
```

Comment:
This Error element will be useful when we remove LicenseUrl from this targets file later.